lib/src/router/interfaces.dart

```diff
@@ -96,21 +96,29 @@ abstract class CanReuse {
 /// if a component can be removed as part of a navigation.
 ///
 /// The `routerCanDeactivate` hook is called with two [ComponentInstruction]s as parameters,
-/// the
-/// first representing the current route being navigated to, and the second parameter
+/// the first representing the current route being navigated to, and the second parameter
 /// representing the previous route.
 ///
 /// If `routerCanDeactivate` returns or resolves to `false`, the navigation is cancelled. If it
-/// returns or
-/// resolves to `true`, then the navigation continues, and the component will be deactivated
+/// returns or resolves to `true`, then the navigation continues, and the component will be deactivated
 /// (the [OnDeactivate] hook will be run) and removed.
 ///
 /// If `routerCanDeactivate` throws or rejects, the navigation is also cancelled.
 ///
 /// ### Example
+///
 /// ```dart
-/// // {@disabled-source "router/ts/can_deactivate/can_deactivate_example.ts" region="routerCanDeactivate"}
+/// // {@source "docs/router/lib/crisis_center/crisis_detail_component.dart" region="routerCanDeactivate"}
+/// @override
+/// /*FutureOr<bool>*/ routerCanDeactivate(next, prev) =>
+///     crisis == null || crisis.name == name
+///         ? true
+///         : _dialogService.confirm('Discard changes?');
 /// ```
+///
+/// See the [router documentation][router] for details.
+///
+/// [router]: docs/guide/router/5#candeactivate
```

Comment:
The link is correct, and the leading `docs/...` is a trigger for use of `https://webdev.dartlang.org/angular`.

There are pros and cons. Pros:
- The docs have been moving around so the `docs` alias gives us one level of indirection.
- I.e., if the docs site moves, then we only need to change the path to the `docs` in one place.
- It is far shorter to type :).

Cons:
- It is "magic"
- Other tools can't follow the link.

Its your call.

If this is the only issue, can you accept this PR and then we can make a sweeping change? (If you could create an issue for that.)